java/cdk8s-core/README.md

```diff
@@ -7,3 +7,6 @@ Example usage of core Kubernetes resources and how to define them within a cdk8s
 - `KubeService` (Core)
 - `KubeDeployment` (Core)
 
+### Setup
```

Comment:
Happy to add this but we should be consistent across all examples then.

Reply:
Ack. Will make the updates in other projects.